app/organization/members/index/route.js

```diff
@@ -3,20 +3,24 @@ import Ember from 'ember';
 export default Ember.Route.extend({
   model(){
     let organization = this.modelFor('organization');
-    return organization.get('users');
+
+    return Ember.RSVP.hash({
+      organization: organization,
+      users: organization.get('users')
```

Comment:
If you make this `model`, you can change the `setupController` to be:

```javascript
setupController(controller, hash) {
  Ember.setProperties(controller, hash);
}
```

Not sure that is enough to worry about though..